docs/registries.conf.5.md

```diff
@@ -0,0 +1,26 @@
+% registries.conf(5) System-wide registry configuration file
+% Brent Baude
+% Aug 2017
+
+# NAME
+registry.conf - Syntax of System Registry Configuration File
+
+# DESCRIPTION
+The REGISTRIES configuration file is a system-wide configuration file for container image
+registries. The file format is TOML.
+
+# FORMAT
+The [TOML_format][toml] is used to build simple list format for registries under two
+categories: `registries` and `insecure_registries`. You can list multiple registries using
```

Comment:
the naming thing could be avoided if we didn't use registries as a toml heading.  then you can have three key value pairs.

registries = []
insecure_registries = []
....

I know this is a bit against the TOML approach which likes headers, but the header is someone benign here and not used.